javascript/array-reverse/README.md

```diff
@@ -0,0 +1,26 @@
+# Challenge Title
+<!-- Description of the challenge -->
+Write a function called reverseArray which takes an array as an argument. Without utilizing any of the built-in methods available to your language, return an array with elements in reversed order.
+
+## Whiteboard Process
+<!-- Embedded whiteboard image -->
+![whiteboard](../assets/whiteboard%20CodeChallange%2001.jpg)
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+Talked with classmates, ChatGPT, old code, chatGPT, google. 
```

Comment:
Hey David! Great job with your first whiteboard! This section is going to be important during interviews. This is going to help the interviewer see how you think through and problem solve.